### DISCUSSION TITLE
Taxes outside EU

### DISCUSSION THREAD

**Discussion by yoxud** [saleor/saleor#7229] [General]
I'm having a problem understanding tax configuration. I am using version 3.0 and located in Canada so I have not setup any vatlayer. I will only sell products in Canada and not abroad. How can I  manually set a fixed tax rate on my products?

**Comment by krzysztofwolski** [comment]
General tax configuration: https://docs.saleor.io/docs/3.0/dashboard/configuration/taxes
Product tax configuration: https://docs.saleor.io/docs/3.0/dashboard/catalog/products#step-2-define-pricing
